### DISCUSSION TITLE
Blur not working after updating. I changed the config to the new decoration { blur{ syntax but its not working

### DISCUSSION THREAD

**Discussion by Sohail-XD** [hyprwm/Hyprland#2952] [Q&A]
decoration {                                                                                                                                                                         
    │ # See https://wiki.hyprland.org/Configuring/Variables/ for more                                                                                                                  
    │ blur {                                                                                                                                                                           
    │ │ │ blur=true                                                                                                                                                                    
    │ │ │ ignore_opacity=true                                                                                                                                                          
    │ │ │ new_optimizations=true                                                                                                                                                       
    │ │ │ size = 0.6                                                                                                                                                                   
    │ │ │ passes = 3                                                                                                                                                                   
    │ │ }
           (other decoration)
}

**Comment by MightyPlaza** [comment]
enabled

**Reply by Sohail-XD** [reply]
Thank you. It was also the blur size I forgot to change lol. I read the wiki and was scratching my head for why it was not working.
